docs/io.cozy.rules.md

```diff
@@ -0,0 +1,37 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Rules doctype
+
+## `io.cozy.rules`
+
+This doctype represents rules, used to target documents in the Cozy.
+
+- `_id`: {string} - The fixed rule ID, that should respect this nomenclature: `<editor-reverse-domain-name>.rules/<rule-name>`. For instance: `io.cozy.rules/payslips`.
+- `description`: {string} - The rule description.
+- `doctype`: {string} - The doctype of the documents targeted by the rule.
+- `rule`: {object}
+  - `limit`: {number} - The maximum number of documents to retrieve with this rule.
+  - `selector`: {object} - The selector to target documents, following the [Mango syntax](https://docs.couchdb.org/en/stable/api/database/find.html#find-selectors)
+
+Note the rule attributes, i.e. `limit` and `selector` can be parametrised by using a `{var-name}` notation.
```

Comment:
```suggestion
Note the rule attributes, i.e. `limit` and `selector` can be parametrised by using a `{var-name}` notation.
```

I don't understand what this means 🤔

Reply:
The rule is expressed as a mango query, that could be expressed with params, like`{date}` here: 
```
"cozyMetadata.createdAt": {
    "$lt": {date}
}
```
I'll try to clarify this

Reply:
Clarification added with 9c99aa5